gcp/docker/example_config.json

```
@@ -39,12 +39,12 @@
                 }
             ]
         },
-        "jobDir": "gs://drobison-gcp-gtc-2020/training_output",
+        "jobDir": "gs://[YOUR STORAGE BUCKET]/training_output",
         "masterConfig": {
-            "imageUri": "gcr.io/k80-exploration/gcp_rapids_training:latest",
+            "imageUri": "gcr.io/[YOUR GCR PATH]/rapids_training_container:10.1",
```

```suggestion
            "imageUri": "gcr.io/[YOUR GCR PATH]/gcp_rapids_training:latest",
```
To match the readme and couple other config files

---

Actually, let me update the README, rather than this; if we're going to move toward a single container instance.